wrappers/wrapper.rb

```diff
@@ -1481,7 +1484,10 @@ def simplify_service_setup_duration_and_vehicle_setup_modifiers(vrp, result = ni
                       } || vrp.services.any?{ |s| s.activity.nil? }
 
         vehicles_grouped_by_matrix_id = vrp.vehicles.group_by(&:matrix_id)
-        vrp.services.group_by{ |s| s.activity.point }.each{ |point, service_group|
+        grouped_services = vrp.services.group_by{ |s| s.activity.point }
+        return nil if grouped_services.map{ |point, _s_g| point.matrix_index }.uniq.size != grouped_services.size
```

Comment:
We should add a TODO (and mention this location in the ticket that I just mentioned) so that we remove this logic once the optim-ortools is based on point_id as well.